### ISSUE TITLE
Review command-line options implementation

### ISSUE THREAD

**Issue by berendo** [sfranusic/test-photo-generator#1]
I think that while the script takes command-line options for things like the destination folder to deposit generated images into, there are places where it appears to (subject to my limited Ruby-fu) still used hard-coded paths like `./test_photos/temp`.

**Comment by sfranusic** [comment]
You are correct. It looks like when I made this the folder option is for where the script pulls files from, but not where it deposits them to... 

Would it be useful to be able to change the destination folder?

**Comment by berendo** [comment]
> Would it be useful to be able to change the destination folder?

Absolutely. And, as far as "pulling files from" (e.g. a source folder), I'm not sure I understand what the purpose of such a "folder" argument is (because I'm making the assumption that the script is pulling a <i>single</i> photo and cloning it. It may make more sense to have an argument for "source photo" that (optionally) provides the path to a single photo to use as the source of the cloning.
